stylesheets/components/Inputs/Input.css

```diff
@@ -0,0 +1,177 @@
+
+
+/* Input Field */
+.input--field {
+  height: 56px;
+  width: 344px;
```

Comment:
instead of a fixed height, you can use padding to adjust the height of the element. Also, instead of giving it a fixed width, use min/max width, so that the content doesn't overflows